Support/Scripts/run-lldb-tests.sh

```diff
@@ -226,7 +226,7 @@ if [[ "${PLATFORM-}" = "1" ]]; then
     args+=("--excluded" "$blacklist_dir/ds2/platform.blacklist")
   fi
 
-  server_args=("p" "--server" "--listen" "$server_port")
```

Comment:
I think I remember this. lldb-server does not need the `*:` in gdbserver mode but does need them in platform mode, which is a little bit confusing.